week-3/refactor/format-as-12-hours.js

```diff
@@ -4,3 +4,19 @@
 // Store this expression in a variable and reference it twice in the function in the correct place
 
 // Explain why it makes more sense to store this expression in a variable
+
+function formatAs12Hours(time) {
+    const hours24 = Number(time.slice(0, 2));
+
+    if (hours24 < 0 || hours24 > 23) {
+        return "Invalid hours in time string";   // check vaild in 24hours fromat 
+    } else if (hours24 === 0) {
+        return "midnight";
```

Comment:
This wouldn't be an exspected output - if you are exspecting hours:minutes midnight would be an annoying output